<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUploadDocument</class>
 <widget class="QMainWindow" name="AdminUploadDocument">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
border:none;
background-color:transparent;
color:#fff;
}



#label
{
color:#fff;

}

#label_2
{
color:#071e26;

}

#centralwidget{
	
	background-color: rgb(75, 75, 75);
}

#sidemenu{
background-color: rgb(171, 171, 171);
border-radius:20px;
}

QPushButton{
padding: 10px;
background-color: rgb(75, 75, 75);
border-radius: 5px;

}

#menuButton{
	background-color: rgb(171, 171, 171);
padding: 10px;
border-radius: 5px;

}

#mainbody{
background-color:071e26;
border-radius:10px;
}

#widget_4{
background-color: rgb(171, 171, 171);
border-radius:20px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>My Desktop Application</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1080</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="mainbody">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1050</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QFrame" name="widget_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>490</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="menu_btn_users_request">
               <property name="text">
                <string>Approved User Request</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/users.svg</normaloff>:/icons/users.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="menu_btn_upload_document">
               <property name="text">
                <string>Upload Document</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>:/icons/upload.svg</normaloff>:/icons/upload.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="sidemenu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>500</width>
              <height>490</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>2000</width>
              <height>490</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>91</width>
               <height>41</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Upload File</string>
             </property>
            </widget>
            <widget class="QPushButton" name="SubmitBtn">
             <property name="geometry">
              <rect>
               <x>630</x>
               <y>230</y>
               <width>211</width>
               <height>61</height>
              </rect>
             </property>
             <property name="text">
              <string>Upload Document</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/icons/upload.svg</normaloff>:/icons/upload.svg</iconset>
             </property>
            </widget>
            <widget class="QPushButton" name="uploadFileBtn">
             <property name="geometry">
              <rect>
               <x>150</x>
               <y>10</y>
               <width>211</width>
               <height>61</height>
              </rect>
             </property>
             <property name="text">
              <string>Upload a File</string>
             </property>
            </widget>
            <widget class="QSplitter" name="splitter_4">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>100</y>
               <width>561</width>
               <height>371</height>
              </rect>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color: black;
}</string>
               </property>
               <property name="text">
                <string>Teachers</string>
               </property>
              </widget>
              <widget class="QListWidget" name="TeachersList">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: black;
</string>
               </property>
              </widget>
              <widget class="QPushButton" name="teachersToggle">
               <property name="text">
                <string>Toggle for All Teachers</string>
               </property>
              </widget>
             </widget>
             <widget class="QSplitter" name="splitter_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color: black;
}</string>
               </property>
               <property name="text">
                <string>Students</string>
               </property>
              </widget>
              <widget class="QListWidget" name="StudentsList">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: black;
</string>
               </property>
              </widget>
              <widget class="QPushButton" name="studentsToggle">
               <property name="text">
                <string>Toggle for All Students</string>
               </property>
              </widget>
             </widget>
             <widget class="QSplitter" name="splitter_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	color: black;
}</string>
               </property>
               <property name="text">
                <string>Session</string>
               </property>
              </widget>
              <widget class="QListWidget" name="SessionsList">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: black;
</string>
               </property>
              </widget>
              <widget class="QPushButton" name="sessionToggle">
               <property name="text">
                <string>Toggle for All Sessions</string>
               </property>
              </widget>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
